src/structures/Sticker.js

```diff
@@ -72,16 +105,62 @@ class Sticker extends Base {
     return new Date(this.createdTimestamp);
   }
 
+  /**
+   * Whether this sticker is partial
+   * @type {boolean}
+   * @readonly
+   */
+  get partial() {
+    return 'username' in this;
+  }
+
+  /**
+   * The guild that owns this sticker
+   * @type {?Guild}
+   * @readonly
+   */
+  get guild() {
+    return this.client.guilds.resolve(this.guildID);
+  }
+
   /**
    * A link to the sticker
-   * <info>If the sticker's format is LOTTIE, it returns the URL of the Lottie json file.
-   * Lottie json files must be converted in order to be displayed in Discord.</info>
+   * <info>If the sticker's format is LOTTIE, it returns the URL of the Lottie json file.</info>
    * @type {string}
    */
   get url() {
-    return `${this.client.options.http.cdn}/stickers/${this.id}/${this.asset}.${
-      this.format === 'LOTTIE' ? 'json' : 'png'
-    }`;
+    return this.client.rest.cdn.Sticker(this.id, this.format);
+  }
+
+  /**
+   * Fetches this sticker.
+   * @returns {Promise<?Sticker>}
+   */
+  async fetch() {
+    const data = await this.client.api.stickers(this.id).get();
+    this._patch(data);
+    return this;
+  }
+
+  /**
+   * Fetches the pack this sticker is part of from Discord, if this is a Nitro sticker.
+   * @returns {Promise<?StickerPack>}
+   */
+  async fetchPack() {
+    return (this.packID && (await this.client.fetchPremiumStickerPacks()).get(this.packID)) ?? null;
```

Comment:
```suggestion
    return this.packID ? (await this.client.fetchPremiumStickerPacks()).get(this.packID) ?? null : null;
```

Reply:
whats wrong with `this.packID && (...)`?

Reply:
I never said something's wrong with it, but I don't know if that's the style we wanna go with everywhere... @iCrawl @kyranet thoughts?

Reply:
Generally I would say this is more readable:
`(this.packID && (await this.client.fetchPremiumStickerPacks()).get(this.packID)) ?? null;`
than this:
`this.packID ? (await this.client.fetchPremiumStickerPacks()).get(this.packID) ?? null : null;`

Reply:
I prefer the current code's approach.